geanyvc/src/geanyvc.c

```diff
@@ -1586,17 +1591,25 @@ vccommit_activated(G_GNUC_UNUSED GtkMenuItem * menuitem, G_GNUC_UNUSED gpointer
 	gtk_paned_set_position(GTK_PANED(vpaned2), height * 50 / 100);
 
 #ifdef USE_GTKSPELL
-	speller = gtkspell_new_attach(GTK_TEXT_VIEW(messageView), NULL, &spellcheck_error);
+    #if GTK_CHECK_VERSION (3, 0, 0)
+        speller = gtk_spell_checker_new ();
+        gtk_spell_checker_attach (speller, GTK_TEXT_VIEW (messageView));
+    #else
+        speller = gtkspell_new_attach(GTK_TEXT_VIEW(messageView), NULL, &spellcheck_error);
+    #endif
 	if (speller == NULL)
 	{
-		ui_set_statusbar(FALSE, _("Error initializing spell checking: %s"),
+        if (spellcheck_error != NULL)
```

Comment:
you could simply check this in place of `speller != NULL` actually.  Though in practice for GtkSpell3 I guess it's totally impossible for `speller` to be `NULL`